packages/envd/internal/host/mmds.go

```diff
@@ -22,17 +22,19 @@ const (
 )
 
 type MMDSOpts struct {
-	TraceID    string `json:"traceID"`
-	InstanceID string `json:"instanceID"`
-	EnvID      string `json:"envID"`
-	Address    string `json:"address"`
+	TraceID      string `json:"traceID"`
+	InstanceID   string `json:"instanceID"`
+	EnvID        string `json:"envID"`
+	Address      string `json:"address"`
+	EventProxyIP string `json:"eventProxyIP"`
 }
 
-func (opts *MMDSOpts) Update(traceID, instanceID, envID, address string) {
+func (opts *MMDSOpts) Update(traceID, instanceID, envID, logsCollectorAddress, eventProxyIP string) {
```

Comment:
```suggestion
func (opts *MMDSOpts) Update(traceID, instanceID, envID, address, eventProxyIP string) {
```
for future us: let's rename it back to `address` when logs collector will live on sandbox http server